core/workflow/iteratorrole.go

```diff
@@ -247,6 +247,13 @@ func (i *iteratorRole) GetName() string {
 	return i.template.GetName()
 }
 
+func (i *iteratorRole) GetDescription() string {
+	if i == nil || i.template == nil {
+		return ""
+	}
+	return i.template.GetDescription()
+}
+
```

Comment:
We don't need this part any more. You can remove the description field from "role" classes.